tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/TinkerGraphWorld.java

```diff
@@ -106,6 +106,7 @@ public GraphTraversalSource getGraphTraversalSource(final LoadGraphWith.GraphDat
 
         private static final List<String> TAGS_TO_IGNORE = Arrays.asList(
                 "@StepDrop",
+                "@StepInject",
```

Comment:
Why? Won't this cause a lot of tests (i.e. Comparability/Orderability) to get ignored?

Reply:
Those are for OLAP tests and OLAP doesn't support `inject()` (among other things)